# Topic names
state_topic: "/ur3_cartesian_velocity_controller/ee_state"
command_topic: "/ur3_cartesian_velocity_controller/command_cart_vel"

# Static gain
Dx: 50
Dy: 50
Dz: 50
Kx: 10
Ky: 10
Kz: 10

# Flag to check for robot limits
check_robot_limits: true

# Self collision limits
self_collision_limit: 0.23
z_limit: 0.23

# Overextension limits
overextension_limit: 0.53

# rotation
theta: 0

# Consecutive control points distance
consecutive_points_distance: 0.2